pydantic/json_schema.py

```diff
@@ -632,8 +632,8 @@ def tuple_variable_schema(self, schema: core_schema.TupleVariableSchema) -> Json
         """
         # NOTE: The `items_schema` is always added, even when we explicitly create a
         # tuple variable schema without an `items_schema`.
-        items = self.generate_inner(schema['items_schema'])  # type: ignore
-        json_schema: JsonSchemaValue = {'type': 'array', 'items': items}
+        items_schema = {} if 'items_schema' not in schema else self.generate_inner(schema['items_schema'])
+        json_schema = {'type': 'array', 'items': items_schema}
```

Comment:
This is my bad. I broke it just before the release here: https://github.com/pydantic/pydantic/pull/5934/files#r1215343647

We can just revert it
```suggestion
        json_schema: JsonSchemaValue = {'type': 'array', 'items': {}}
        if 'items_schema' in schema:
            json_schema['items'] = self.generate_inner(schema['items_schema'])
```

Reply:
oh, I just noticed it's the same code 👀 

Anyway... suggestion still applies as it looks more readable

Reply:
The new version is the same as the list version. Let’s try to avoid delaying work for minor things like this.

Reply:
I don't see how a suggestion would make that effect...

Reply:
This could have been merged yesterday. In these situations I usually like to approve *and* leave nits/suggestions so the author can decide if they want to just merge, fix a couple and merge, or fix all them and merge. But we avoid a 24 hour delay of another round of review.